run/helm/snap/README.md

```diff
@@ -0,0 +1,246 @@
+
+| Parameter                 | Description                                         | Default                           |
+|---------------------------|-----------------------------------------------------|-----------------------------------|
+| `image.repo`              | docker image                                        | intelsdi/snap4kube                |
+| `image.tag`               | docker image tag                                    | latest                            |
+| `image.pullPolicy`        | image pull policy                                   | IfNotPresent                      |
+| `mounts`                  | list of volumes to mount inside the Snap container  | /proc:/proc_host                  |
+| `host_autoload`           | host path to the directory with plugins and tasks   | /opt/autoload                     |
+| `privileged`              | security context needed to access files on the host | false                             |
+| `config.log_level`        | log level of Snap, (1=debug, 5=error)               | 3                                 |
+| `config.log_path`         | container path to Snap logs                         | /snap                             |
+| `config.autodiscover_path`| list of volumes to mount inside the Snap container  | /opt/snap/autoload                |
+| `autoload`                | list of Snap plugins/tasks URLs to download and load| empty                             |
+
+## 7. Loading Snap plugins
+
+The easiest way to download and load Snap plugins is to specify plugins binaries URL in the `autoload` parameter in `values.yaml` file.
+
+Here is a simple example of using [CPU collector plugin](https://github.com/intelsdi-x/snap-plugin-collector-cpu) with [file publisher plugin](https://github.com/intelsdi-x/snap-plugin-publisher-file).
+
+```
+$ cat <<EOF >> new_values.yaml
+autoload:
+  - https://github.com/intelsdi-x/snap-plugin-collector-cpu/releases/download/6/snap-plugin-collector-cpu_linux_x86_64
+  - https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64
+  - https://raw.githubusercontent.com/intelsdi-x/snap-integration-kubernetes/master/examples/tasks/cpu-file.json
+EOF
+```
+
+With the command above we override default values adding list of URLs of two plugin binaries `snap-plugin-collector-cpu_linux_x86_64` and `snap-plugin-publisher-file_linux_x86_64`.
+The third URL downloads exemplary task for CPU plugin.
+
+Now running command:
+
+`$ helm install -f new_values.yaml stable/snap`
+
+will create your very own Snap release!
+
+Now you can list all the pods in order to verify if Snap pods are running correctly:
+
+`$ kubectl get pods --all-namespaces`
+
+The output should contain Snap pods:
+
+```
+NAMESPACE     NAME                                    READY     STATUS    RESTARTS   AGE
+default       snap-1kc54                              1/1       Running   0          1m
```

Comment:
The purpose of this Helm chart was to deliver bare Snap deployable on Kubernetes (no plugins, no tasks, only examples how to run). But of course we can go beyond the original goal and transform it into monitoring stack.